six/modules/c++/six/source/NITFHeaderCreator.cpp

```diff
@@ -108,7 +108,7 @@ NITFHeaderCreator::NITFHeaderCreator() :
     loadXmlDataContentHandler();
 }
 
-NITFHeaderCreator::NITFHeaderCreator(mem::SharedPtr<Container> container) : 
+NITFHeaderCreator::NITFHeaderCreator(mem::SharedPtr<Container> container) :
```

Comment:
Are we using mem::SharedPtr here to match an existing interface? If not we should use std::shared_ptr

Reply:
Basically, yes. There's a whole call hierarchy of `mem::SharedPtr` going on here. I think doing a separate PR that's kind of a general C++11 update pass (or at least just the smart pointers) would make more sense than making piecemeal changes.